dojo/settings/settings.dist.py

```diff
@@ -1744,6 +1748,13 @@ def saml2_attrib_map_format(dict):
 # see https://github.com/laymonage/django-jsonfield-backport
 SILENCED_SYSTEM_CHECKS = ["django_jsonfield_backport.W001"]
 
+# By default, this mapping is not configured (set to None). If configured, it allows
+# the "Problems" button to appear in Dojo's left toolbar.
+# You can check more information in https://homepages.dcc.ufmg.br/~leonardooliveira/defectdojo/README.md
+# A finding-to-problem mapping covering Nmap, OpenVAS and Nuclei is available in
+# <link to JSON file>
```

Comment:
I actually meant that we should have a link to the existing JSON file here so people can copy-paste it in if they want.